distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email.enso

```diff
@@ -0,0 +1,41 @@
+import project.Any.Any
+import project.Data.Text.Text
+import project.System.File.File
+import project.Data.Vector.Vector
+import project.Network.Email.Email_Address.Email_Address
+import project.Network.Email.Email_Body.Email_Body
+import project.Network.Email.Email_Object.Email_Object
+import project.Network.Email.Email_Provider.Email_Provider
+import project.Network.Email.Email_Attachment.Email_Attachment
+import project.Network.Email.Email_Send_Response.Send_Response
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Text_Input, Vector_Editor, Single_Choice
+import project.Data.Pair.Pair
+import project.Metadata.Widget
+import project.Metadata.Display
+
+## ---
+   aliases: [mail]
+   group: Web
+   icon: email
+   ---
+   Sends an email.
+@to default_widget
+@cc default_widget
+@bcc default_widget
+@body Email_Body.default_widget
+send from:Email_Address (to:Vector Email_Address)=[] (cc:Vector Email_Address)=[] (bcc:Vector Email_Address)=[] (subject:Text)="" (body:Email_Body)="" (attachments:Vector Email_Attachment)="" (provider:Email_Provider) =
+    _ = [provider]
+    email_object = Email_Object.Value from to cc bcc subject body attachments
+    provider.send email_object
+
+## ---
+    private: true
+    ---
+default_widget (display : Display = ..When_Modified) -> Widget =
```

Comment:
- Move `private` methods outside of API
- either move them to `private `module
- or make them `private` with a keyword